src/typings/math.d.ts

```diff
@@ -0,0 +1,2 @@
+declare type Vector2 = [x: number, y: number]
+declare type Interval = [min: number, max: number]
```

Comment:
Currently these ambient types aren't getting included in the rollup. Trying to find a way to fix that.